wgengine/userspace.go

```diff
@@ -112,6 +111,8 @@ type userspaceEngine struct {
 	trimmedDisco        map[tailcfg.DiscoKey]bool // set of disco keys of peers currently excluded from wireguard config
 	sentActivityAt      map[netaddr.IP]*int64     // value is atomic int64 of unixtime
 	destIPActivityFuncs map[netaddr.IP]func()
+	statusBufioReader   *bufio.Reader // reusable for UAPI
+	statusBufioWriter   *bufio.Writer // reusable for UAPI
```

Comment:
I don't think we need a bufio.Writer at all; we can just pass the pipe writer directly without wrapping.

Reply:
Done.